web/app/scripts/test-app/test.es.js

```diff
@@ -20,6 +20,9 @@
 
                 this.fields = FieldsetParserService.parseFields(form);
                 this.model = FieldsetParserService.parseModel(form);
+
+                this.getUrl = 'http://127.0.0.1:8000/forms/1'
+                this.postUrl = 'http://127.0.0.1:8000/forms/1/respond/test-user'
```

Comment:
They are used to populate the survey in `index.html` (if you have the api running)